components/scream/src/physics/p3/p3_functions_f90.hpp

```diff
@@ -1142,6 +1142,57 @@ void p3_main_pre_main_loop_f(
 
 }
 
+///////////////////////////////////////////////////////////////////////////////
+
+struct P3MainMainLoopData
```

Comment:
Minor: MainMain sounds a bit funky. I'm guessing p3_main has a main and a post loop, but still. Would P3MainLoopData work? It would definitely sound better for the avg reader.

Reply:
Done, push coming.